app/helpers/status_updates_helper.rb

```diff
@@ -0,0 +1,7 @@
+module StatusUpdatesHelper
```

Comment:
I wouldn't define this in a module since you're only using it once. You can easily write this inline in `app/views/students/status_updates/_form.html.slim`